### DISCUSSION TITLE
[tsml_next_meetings] short code crashed

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#743] [General]
Update: Took care of some Wordfence issues and updated recaptcha plugin. Re-enabled [tsml_next_meetings] and now it works again. Very strange.

Yesterday I used the [tsml_next_meetings] short code on the dixieaaoffice.com home page and it was working. today the next meetings data table for next meetings was missing. For now I have deleted the [tsml_next_meetings] short code.

We are using the Wordfence security plugin - This morning Wordfence reported several critical errors. I used it to fix one file and delete a couple of others. I'm wondering if there's a connection.

The meetings page still works as expected. Now just waiting to be connected the Meeting guide app.

**Comment by tech2serve** [comment]
@DCO84770, glad you got it working. It often seems interactions with other plugins is the weak link for Wordpress. 

If the problem returns, please try to capture more of the error messages from the console, and perhaps we can look at this closer.
